bindata/network/openshift-sdn/alert-rules.yaml

```diff
@@ -67,7 +67,7 @@ spec:
       annotations:
         message: The average time between iptables resyncs is too high. NOTE - There is some scrape delay and other offsets, 90s isn't exact but it is still too high.
       expr: |
-        time() - (sum(kubeproxy_sync_proxy_rules_last_timestamp_seconds) / :kube_pod_info_node_count:) > 90
+        (time() - (sum(kubeproxy_sync_proxy_rules_last_timestamp_seconds)) / sum(kube_pod_info{namespace="openshift-sdn",  pod=~"ovs.*"})) > 90
```

Comment:
The new parenthesization is weird; before it was `TIME - (SUM / COUNT) > 90`, now it's `(TIME - (SUM) / COUNT) > 90` which I guess is still the same thing because of order of operations but it's weirder.

Reply:
fixed